### ISSUE TITLE
Вылетает на Steam версии

### ISSUE THREAD

**Issue by aka-sektor** [abramcumner/xray15#6]
И этот двиг проблемы не обошли.

При запуске, ловлю:
![default](https://user-images.githubusercontent.com/25452748/42165724-970450f4-7e11-11e8-971c-2142ca171342.png)

[xray_aka_sektor.log](https://github.com/abramcumner/xray15/files/2155304/xray_aka_sektor.log)

**Comment by aka-sektor** [comment]
Попробую с ключом -dsound

**Comment by aka-sektor** [comment]
**UPD4:**
Ок, выбрал новую игру, экран загрузки, потом вылетело.
[xray_aka_sektor.zip](https://github.com/abramcumner/xray15/files/2155388/xray_aka_sektor.zip)

**Comment by abramcumner** [comment]
гоговская не поможет. Залил исправленный скрипт gulag_general.script.
А для звука наверное надо поставить: https://www.openal.org/downloads/oalinst.zip

**Comment by aka-sektor** [comment]
Нету кнопки, чтобы скачать 108, придется вручную

**Comment by redpython** [comment]
В официальном сталкире в /bin лежит OpenAL32.dll, а тут нет. Установщик установит библу в системную папку, откуда её смогут дёргать все сталкиры.

>Нету кнопки, чтобы скачать 108, придется вручную
Установи Github Desktop и у тебя всегда будут свежие исходники.

**Comment by abramcumner** [comment]
> В официальном сталкире в /bin лежит OpenAL32.dll, а тут нет

openal32.dll как раз есть, она из исходников собирается. Наверное wrap_oal.dll не хватает.

**Comment by aka-sektor** [comment]
`для звука наверное надо поставить`
Действительно, OpenAL не был установлен. Теперь понятно, с чем ошибка связана.

`исправленный скрипт gulag_general.script.`
А сам не тестил что ли?

`Установи Github Desktop`
Так к себе на вики ссылки закинуть надо тоже, видимо не сразу новые архивы собирает.

В общем, игра запустилось, но по выходе все равно ошибка, приходится нажимать закрыть приложение.

**Comment by abramcumner** [comment]
>Нету кнопки, чтобы скачать 108, придется вручную

Через час появится. Там движок заново собирается :)

**Comment by abramcumner** [comment]
> А сам не тестил что ли?
На чистой игре нет, с боржчем запускаю.

**Comment by aka-sektor** [comment]
А номер версии в движке что ли? В менюхе 107.
Вроде ж по жизни в скрипте было меню.
ОГСЕ там ещё, в конфиги сделали помню.

**Comment by abramcumner** [comment]
107 берется из движка по номеру сборки на аппвейор.

> Вроде ж по жизни в скрипте было меню.
У тебя же сейчас оригинальные конфиги и скрипты(с минимальными изменениями), так что по жизни было, как сейчас :)

**Comment by aka-sektor** [comment]
Так что по ошибке на выходе?

**Comment by abramcumner** [comment]
Не знаю, вечером посмотрю.

**Comment by redpython** [comment]
>openal32.dll как раз есть, она из исходников собирается. Наверное wrap_oal.dll не хватает.

Точно. Удалил эту библу из C:\Windows\SysWOW64 - и при запуске стала вылезать ошибка "can't create sound device". Причём движок запускал x86.

**Comment by abramcumner** [comment]
Добавлю в папку бин, тогда не надо будет openal устанавливать.
